checkout/checkin.md

```diff
@@ -55,6 +57,7 @@ Content-Type: application/json
 {
     "token": "7e380fbb3196ea76cc45814c1d99d59b66db918ce2131b61f585645eff364871",
     "operations": [
+        // Deprecated operation. Do not use!
```

Comment:
Can we have some information on why `redirect-consumer-identification` is deprecated?

Reply:
We only support using the hosted version of checkin. We dont want to anyone to accidentally attempt to implement the redirect version and think it would be best to mark it as deprecated.

Reply:
@marinafrydenb, can you please add a task to document that `redirect-consumer-identification` should be marked as deprecated?